macros/ano-20macros.cfg

```diff
@@ -502,6 +502,11 @@
         {KILLEDBY (Robert the Hefty) (Reme Carrenemoe)}
         {MSGm_Reme _"This is for your outlaws' attack on the Carrenemoe clan, and my capture!"}
         {MSGa_Gawen _"Wait, Reme, I had some questions I wanted to ask him!"}
+        [kill]
```

Comment:
This should go before Gawen's line.

Reply:
oh yeah I was about 50/50 as to whether it should go before or after; I'll move it

Reply:
moved in 61ba872